Dockerfile

```diff
@@ -178,6 +178,7 @@ RUN zypper addrepo http://download.opensuse.org/repositories/home:illuusio/openS
     /usr/share/emacs/ \
     /usr/share/xemacs/ \
     /usr/share/locale/*/LC_MESSAGES/*.[mp]o* \
+    /var/log/ \
```

Comment:
This removes the directory as well right? Are we sure this doesn't create problems with future logging activity?